### DISCUSSION TITLE
WARNING: FAKE HOMEBREW WEBSITE

### DISCUSSION THREAD

**Discussion by ngarantBeauvoir** [Homebrew/discussions#5351] [Everyday usage]
### Output of `brew config`

```shell
0
```

### Output of `brew doctor`

```shell
0
```

### Description of issue

Hi!
I hope this warning will be seen by many.
There is a sponsored link on google when you search homebrew, it naturally show on the first line, making it extremely easy to click. The page looks exactly like [brew.sh](brew.sh), but the language selector is missing, and the command to install homebrew is replaced by "Download for MAC". The website is **https://hornebrew.mom/**.

Please report this website, and hopefully it will be taken down.

Thanks

**Comment by miccal** [comment]
https://github.com/Homebrew/brew.sh/issues/1022
